pkg/blob/blob.go

```diff
@@ -266,20 +266,20 @@ func (d *Driver) Run(endpoint, kubeconfig string, testBool bool) {
 }
 
 // GetContainerInfo get container info according to volume id, e.g.
-// input: "rg#f5713de20cde511e8ba4900#containerName#uuid"
-// output: rg, f5713de20cde511e8ba4900, containerName
-// input: "rg#f5713de20cde511e8ba4900#containerName#uuid#namespace"
-// output: rg, f5713de20cde511e8ba4900, containerName, namespace
-func GetContainerInfo(id string) (string, string, string, string, error) {
+// input: "subsID#rg#f5713de20cde511e8ba4900#containerName#uuid"
```

Comment:
add subsID to the beginning of the volumeID would be a breaking change, suggest appending subsID to the original volumeID format, and add ut test to check original format works 

also if subsID is same as cluster, we should keep the same format, it's not necessary to always append subsID to the volumeID since in most cases, it's in same subs.

Reply:
Good point. How about I use $ instead of #, that is to say add `subsID$` to the beginning of the volumeID? That should be backward compatible.